Helper/Data.php

```diff
@@ -32,23 +51,54 @@ public function __construct(
     }
 
     /**
-     * @return bool Indicates if 2FA can be bypassed for admin access
+     * Indicates if 2FA can be bypassed for admin access
+     *
+     * @return bool
      */
     public function getBypassAdmin(): bool
     {
         return $this->isBypassAllowed(self::ENV_VARIABLE_BYPASS_2FA_ADMIN);
     }
 
     /**
-     * @return bool Indicates if 2FA can be bypassed for API token generation
+     * Indicates if 2FA can be bypassed for API token generation
+     *
+     * @return bool
      */
     public function getBypassAPI(): bool
     {
         return $this->isBypassAllowed(self::ENV_VARIABLE_BYPASS_2FA_API);
     }
 
+    /**
+     * Validate the usernames that can be bypassed
+     *
+     * @param string $userName
+     * @return bool
+     */
+    public function isAllowedBypassAPiByUsername(string $userName): bool
```

Comment:
For consistency with other functions, can the `i` in API be made uppercase?

Reply:
This is fixed